Chap_API_Struct.tex

```diff
@@ -99,24 +99,31 @@ \subsection{PMIx Return Status Constants}
 \label{api:struct:errors}
 \declarestruct{pmix_status_t}
 
-The \refstruct{pmix_status_t} structure is an \code{int} type for return status. The tables shown in this section define the possible values for \refstruct{pmix_status_t}.
-PMIx errors are required to always be negative, with \code{0} reserved for \refconst{PMIX_SUCCESS}. Values in the list that were deprecated in later standards are denoted as such. Values added to the list in this version of the standard are shown in \textbf{\color{magenta}magenta}.
+The \refstruct{pmix_status_t} type is an \code{int} compatible value for return status values.
+\ac{PMIx} return values other than \refconst{PMIX_SUCCESS} are required to always be negative.  The return status value for a successful operation is
+\refconst{PMIX_SUCCESS}, which must have an integer value of 0:
+ 
+\begin{constantdesc}
+\declareconstitem{PMIX_SUCCESS}
+Success.
+\end{constantdesc}
 
 \adviceimplstart
-A PMIx implementation must define all of the constants defined in this section, even if they will never return the specific value to the caller.
+A \ac{PMIx} implementation must define all of the return status constants defined in the \ac{PMIx} standard, even if the implementation will never return the specific value to the caller.
 \adviceimplend
 
 \adviceuserstart
-Other than \refconst{PMIX_SUCCESS} (which is required to be zero), the actual value of any \ac{PMIx} error constant is left to the \ac{PMIx} library implementer. Thus, users are advised to always refer to constant by name, and not a specific implementation's value, for portability between implementations and compatibility across library versions.
+Other than \refconst{PMIX_SUCCESS} (which is required to be zero), the integer value of any \ac{PMIx} error constant is left to the \ac{PMIx} library implementer with the constraint that it be negative and greater than
+\refconst{PMIX_EXTERNAL_ERR_BASE}.
```

Comment:
You may need to clarify here - `greater than` a negative number mathematically means a negative number whose absolute value is less. In this case, we want a negative number whose absolute value is _greater_ than the base.

Reply:
sure, that sounds fine